src/TreeMap/App.svelte

```diff
@@ -256,99 +257,97 @@
       </div>
     </div>
     <div class="mapContainer">
-        {#if community === "true"}
-           {#if theme === "light"}
-            <div id="map" class="view" use:createMap />
-           {:else}
-             <div id="map" class="view" use:createMap />
-           {/if}
+      {#if community === "true"}
+        {#if theme === "light"}
+          <div id="map" class="view" use:createMap />
         {:else}
-          {#if theme === "light"}
-           <div id="map" class="view" use:createMap />
-          {:else}
-            <div id="map" class="view" use:createMap />
-          {/if}
+          <div id="map" class="view" use:createMap />
+        {/if}
+      {:else if theme === "light"}
+        <div id="map" class="view" use:createMap />
+      {:else}
+        <div id="map" class="view" use:createMap />
+      {/if}
```

Comment:
@sarveshpro Was this done by some automated process of your IDE? Which tool are you using? I'd like to properly format our code, but in this case it does not show the four use cases we have to do for loading the map so clearly anymore as before with 
```javascript
        {#if community === "true"}
           {#if theme === "light"}
            <div id="map" class="view" use:createMap />
           {:else}
             <div id="map" class="view" use:createMap />
           {/if}
        {:else}
          {#if theme === "light"}
           <div id="map" class="view" use:createMap />
          {:else}
            <div id="map" class="view" use:createMap />
          {/if}
        {/if}
```